### PR TITLE
change the dist react components to be in plain js becasue that is mo…

### PR DESCRIPTION
**Pull request by walfly**
…re expected behavior from a library

Keeping the jsx files so its not breaking

### CODE REVIEW

**Comment by twhid** [comment]
@walfly is going to make this all sync, then we'll merge.

**Comment by walfly** [comment]
Mostly synchronous I didn't want to change the html to react function.

File: generateDist.js [hunk 1]
```diff
@@ -16,29 +17,37 @@ const checkError = err => {
...
 
-    fs.readFile(leFile, (err, data) => {
-        checkError(err);
+    const contents = fs.readFileSync(leFile, {encoding: 'UTF-8'});
 
-        // generate template string module :
-        const fileContents = data.toString();
-        const stringContents = `module.exports = \`${fileContents}\`;`;
+    // generate template string module :
+    const stringContents = `module.exports = \`${contents}\`;`;
```
Comment (twhid) [inline generateDist.js]: Why not use actual quotes inside the template literal (I understand that this isn't your code, but escaping the template literal ticks seems uselessly complex). I can't think there'd be a reason to do it.

Comment (twhid) [inline generateDist.js]: @walfly just that one thing then I'll merge it.

Comment (walfly) [inline generateDist.js]: i think multiline strings is the reason

Comment (walfly) [inline generateDist.js]: but it shouldn't make a difference either way.

Comment (twhid) [inline generateDist.js]: Ah yeah, I think you're right.

Comment (walfly) [inline generateDist.js]: actually i think it breaks the other way.  you'd have have to escape all strings in the html and add new lines.

**Pull request merged**